omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated :doc:`public group <public>` to host the raw data
+   underlying the publication.
+#. Add all the authors of the paper to this new public group.
```

Comment:
Add additional point here:
``If you did not do so already, configure the public user (link to creating and confuguring the public user) and add the public user as a member of the newly created dedicated group.

Reply:
Added below